user_docs/en/changes.t2t

```diff
@@ -12,6 +12,7 @@ What's New in NVDA
   - Laptop: ``NVDA+ctrl+shift+.``.
   -
 - Introduced new commands to move the review cursor by page where supported by the application. (#14021)
+  - These are especially useful for moving across screens of text, such as paginated output, in Windows Terminal version 1.18 and later.

```

Comment:
I think In-Process may be a better way to communicate this.
As this feature has not been introduced on the terminal side yet, and the 2022.4 translation freeze is over, this PR can no longer make 2022.4